config/linux_gnu.py

```diff
@@ -6,7 +6,7 @@
 
 #
 #flags_arch = '-g -fprofile-arcs -ftest-coverage' # gcov
-flags_arch = '-O3 -g -ffast-math -funroll-loops -fPIC -pedantic'
+flags_arch = '-O1 -g -ffast-math -funroll-loops -fPIC -pedantic'
```

Comment:
Yeah, it was my intention to take the changes from #63 and add them to this PR (and close the other PR - I had worked on it separately to avoid polluting this PR). I'll hopefully do later this afternoon